lib/myfinance/resources/financial_account.rb

```diff
@@ -16,6 +16,22 @@ def create(entity_id, params = {})
           respond_with_object(response, resource_key)
         end
       end
+
+      #
+      # Destroys a payable/receivable account
+      #
+      # [API]
+      #   Method: <tt>DELETE /entities/:entity_id/payable_accounts/:id</tt>
+      #   Method: <tt>DELETE /entities/:entity_id/receivable_accounts/:id</tt>
+      #
+      #   Documentation: https://app.myfinance.com.br/docs/api/payable_accounts#delete_destroy
+      #   Documentation: https://app.myfinance.com.br/docs/api/receivable_accounts#delete_destroy
+      #
+      def destroy(id, entity_id)
+        http.delete(endpoints(id, entity_id, :destroy)) do |response|
```

Comment:
Unused block argument - `response`. You can omit the argument if you don't care about it.